CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.47.0 (2021-06-03)
+
+- Style changes and automatically display vertical nav for `tcs`. [#196](https://github.com/blackbaud/auth-client/pull/196)
```

Comment:
Usually we start the changelog entries with one of a few standard verbs. Would that work here as well?
```suggestion
- Update styles and automatically display the vertical nav for `tcs`. [#196](https://github.com/blackbaud/auth-client/pull/196)
```